server/graphql/v2/mutation/HostApplicationMutations.ts

```diff
@@ -305,6 +305,12 @@ const approveApplication = async (host, collective, req) => {
       transaction,
     });
 
+    // Convert all active tiers to host currency
+    const tiers = await models.Tier.findAll({ where: { CollectiveId: collective.id, deletedAt: null }, transaction });
+    tiers.map(tier => {
+      tier.update({ currency: host.currency }, { transaction });
+    });
+
```

Comment:
A simpler/more performant alternative to that:

```suggestion
    await models.Tier.update({ currency: host.currency }, { where: { CollectiveId: collective.id }, transaction });

```

Reply:
Thanks. Done.